recipe/meta.yaml

```diff
@@ -39,7 +38,7 @@ requirements:
     - arrow-cpp-proc * {{ build_ext }}
     - boost-cpp
     - cython
-    - numpy >=1.16
+    - numpy 1.16.*
```

Comment:
@pearu, it's important to only depend on the oldest supported numpy for building.

PS. This is a comment on commit https://github.com/conda-forge/pyarrow-feedstock/pull/101/commits/7899377d728fa70948b1d20411c01e58a88c8ffc, in case it doesn't show up in the PR thread.

Reply:
OK, makes sense.